controllers/drpolicy_controller.go

```diff
@@ -243,6 +250,36 @@ func s3ProfileValidate(ctx context.Context, apiReader client.Reader,
 	return "", nil
 }
 
+func validateCIDRsFormat(ctx context.Context, drpolicy *ramen.DRPolicy, log logr.Logger) (string, error) {
+	// validate the CIDRs format
+	for i := range drpolicy.Spec.DRClusterSet {
+		cluster := &drpolicy.Spec.DRClusterSet[i]
+		if reason, err := CIDRsFomatValidate(ctx, cluster.CIDRs, log); err != nil {
+			return reason, err
+		}
+	}
+
+	return "", nil
```

Comment:
just return the error message, reason seems to be redundant?

Reply:
removed